backend/src/data/repositories/userRepository.ts

```diff
@@ -15,6 +16,17 @@ class UserRepository extends Repository<User> {
   getById(id: string): Promise<User> {
     return this.findOne(id);
   }
+
+  async editUser(id:string, data: IUserClient): Promise<User> {
+    await this.update(
+      id,
+      data
+    );
+
+    const user = await this.findOne(id);
```

Comment:
findOne should be replaced by getById method
```suggestion
    const user = await this.getById(id);
```

Reply:
After this change, the code will refer to other methods of this class. You are right that the logic of another method is used here, but it seems to me that the code will be cleaner if this one line refers to the methods of the ORM and not the current class.

Reply:
it's ok to use findOne. Because in other case one method calls another that calls another that calls another..